index.js

```diff
@@ -244,6 +244,28 @@ class Mapeo extends events.EventEmitter {
     })
   }
 
+  getDeviceStatus (cb) {
+    this.osm.ready(() => {
```

Comment:
I noticed something looking at this, maybe @noffle can confirm: what happens if an error occurs prior to kappa-osm being ready? From a quick look at https://github.com/digidem/kappa-osm/blob/master/index.js#L40 it looks like the error event is emitted, but the ready functions are just never called. Ideally we would catch an error in this function and callback with any error, rather than just hanging.

Reply:
Yes, I think it'd be really nice if `ready()` could call its `cb` so it's `osm.ready(err => {...})` instead. That'll be a Future Thing though.

Reply:
can we open an issue for this?